### PR TITLE
Feature/edit habit

### PR DESCRIPTION
**Pull request by LoginALee**
## Explain why this change is being made.
As a user with active habits, i would like to be able to edit my habits information so i can change them

## Explain how this is accomplished.
When you are on habits index, you can click on each habit card to go to their show page, where you can edit it

## How do you manually test this?
-Scenario 1 - User is in /habits Given the user clicks on one of the habits card Then the user goes to /habit/:habit_id/ Then the user clicks on edit habit button Then a form appears, and the user fulfills the fields he would like to edit. The user can edit The habit name, The habit description and the Final date

### Screenshots
image

image

image

### CODE REVIEW

File: app/assets/stylesheets/habits.scss [hunk 1]
```diff
@@ -3,8 +3,27 @@
 // You can use Sass (SCSS) here: https://sass-lang.com/
 #calendarContainer,
 #organizerContainer {
-  float: left;
-  margin: 5px;
+  // float: left;
+  // margin: 5px;
```
Comment (jvargas98) [inline app/assets/stylesheets/habits.scss]: please remove these lines if you don't need them

File: app/assets/stylesheets/note.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the note controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (jvargas98) [inline app/assets/stylesheets/note.scss]: please remove this empty file

**Review by jvargas98 — COMMENTED** [review]
I left some comments, great work 💪🏼

**Review by ese-varo — APPROVED** [review]
:rocket:

**Review by jvargas98 — APPROVED** [review]
🚀

**Review by GregVaz — APPROVED** [review]
:rocket:

**Pull request merged**